src/server/pps/server/api_server.go

```diff
@@ -1423,12 +1423,6 @@ func (a *apiServer) validatePipelineRequest(request *pps.CreatePipelineRequest)
 
 // TODO: Implement the appropriate features.
 func (a *apiServer) validateV2Features(request *pps.CreatePipelineRequest) (*pps.CreatePipelineRequest, error) {
```

Comment:
You can just remove this function since we do not need to block any features once these two are removed.

Reply:
@avigil - should probably still do this ^

Reply:
This method (and references to it are removed now)